dist/common/scripts/scylla_raid_setup

```diff
@@ -202,8 +198,16 @@ WantedBy=multi-user.target
     systemd_unit.reload()
     if args.raid_level != '0':
         md_service.start()
-    mount = systemd_unit(mntunit_bn)
-    mount.start()
+    try:
+        mount = systemd_unit(mntunit_bn)
+        mount.start()
```

Comment:
but what will happen if you fail to mount? will the next steps of setup fail?
what if you add here a small loop, with a 1s sleep (like, 5 iterations) to give some time for the availability of the mount point?

Reply:
> but what will happen if you fail to mount? will the next steps of setup fail?

I haven't change the error behavior, when `mount.start()` failed it causes Traceback and setup will fail, just same as what we see today.

Only difference is these additional error message. 
```
        if not os.path.exists(uuidpath):
            print(f'\nERROR: {uuidpath} is not found\n')
        elif not stat.S_ISBLK(os.stat(uuidpath).st_mode):
            print(f'\nERROR: {uuidpath} is not block device\n')
```

Because I rethrow exception in `except SubprocessError` block:
https://github.com/scylladb/scylladb/pull/11666/commits/dac5d7ad6fd64831b9682278960a925f01eccbb1#diff-8aa8fdb174bd6da9e68d0cba6ba42ae0a8564173ae23b6e1dce66db9a422aa95R209

> what if you add here a small loop, with a 1s sleep (like, 5 iterations) to give some time for the availability of the mount point?

Well, we can retry, but var-lib-scylla.mount tries to wait for /dev/disk/by-uuid/$UUID become available until `TimeoutStartSec` == 90sec (default value on systemd.conf).
So it actually already wait for 1.5 min on single iteration, I guess it unlikely become available after that.

And even we executing `udevadm settle` prior to starting var-lib-scylla.mount, it will wait for systemd-udevd to create device file on /dev/disk/by-uuid.
So I think we waited enough for the device availability.


But maybe we can add some workaround code as failsafe:
- sleep for few sec before try mounting the device
 since the problem looks like we need to wait more
- when var-lib-scylla.mount fails, run ` mount $fsdev $mount_at`
 we already suceeded to format the device using real path ($fsdev, user specified path like /dev/nvme0n1 or /dev/sda1), it should able to mount as well

Reply:
@fruch are you okay with this?

Reply:
Looks o.k.
 But seems like it doesn't happens too much anymore, according to:
https://github.com/scylladb/scylladb/issues/11617#issuecomment-1275125162